ui/app/models/database/role.js

```diff
@@ -29,14 +29,14 @@ export default Model.extend({
     editType: 'ttl',
     defaultValue: '1h',
     label: 'Generated credentials’s Time-to-Live (TTL)',
-    subText: 'Vault will use the engine default of 1 hour',
+    helperTextDisabled: 'Vault will use a TTL of 1 hour.',
```

Comment:
Did Ivana have any thoughts on helperTextEnabled for these?

Reply:
No, but it'd be good to clarify a final time when she is back tomorrow.

Reply:
@Monkeychip - Ivana wants to stick with the default text!